### DISCUSSION TITLE
Ability to Hide Ads

### DISCUSSION THREAD

**Discussion by JadeWhite** [modrinth/.github#167] [Feature requests]
Can we add a toggle to the settings page to disable the "Ads via Adrinth" banner at the top of the instance management page?

**Comment by Onako2** [comment]
Modrinth finances with ads. You should rather install an adblocker on curseforge instead of Modrinth

**Comment by bangetto** [comment]
that one ad is really not that annoying, but I mean if you really hate it you can install an adblocker

**Comment by JadeWhite** [comment]
Decided to switch to PrismLauncher after all, interface sucks - but no ads.
